pkg/interface/src/views/components/StatusBar.tsx

```diff
@@ -36,7 +36,9 @@ const StatusBar = (props) => {
   const invites = [].concat(
     ...Object.values(inviteState).map(obj => Object.values(obj))
   );
-  const metaKey = window.navigator.platform.includes('Mac') ? '⌘' : 'Ctrl+';
+  const leapKey: string = useSettingsState(
+    useCallback(s => s.keyboard['leap'], ['leap'])
+    )?.replace('meta+', '⌘') || '';
```

Comment:
Did you want to always show `⌘` here?

Reply:
If it's a meta shortcut, probably. Do you think it should be agnostic for Linux or the like?

Reply:
Yeah I think so, it seems like y'all were using `window.navigator.platform.includes('Mac') ? '⌘' : 'Ctrl+'` before.

Reply:
Yes, but there's no Linux analogue for the Meta key. Previously, we bound both Ctrl and Meta, and showed Ctrl for other platforms. If you set your key to Meta, what would you refer to it as in a Linux context? Just `Super+`?

Reply:
Oh I see, yeah I guess so, `⌘`, `Win` or `Super`